pyannote/audio/interactive/database/recipe.py

```diff
@@ -0,0 +1,94 @@
+import sys
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, Optional
+
+import prodigy
+import yaml
+from prodigy.components.db import connect
+
+from .answers2file import answers2file
+
+
+def comparePath(array_path):
+    minimum_path = min(array_path, key=lambda x: len(x.parts))
+    for i, dir in enumerate(minimum_path.parts):
+        for path in array_path:
+            if not path.parts[i] == dir:
+                i = i - 1
+                break
+        else:
+            continue
+        break
+    return Path(*minimum_path.parts[0 : i + 1])
+
+
+@prodigy.recipe(
+    "pyannote.database",
+    dataset=("Dataset where the annotations are", "positional", None, str),
+    database=("Database name", "positional", None, str),
```

Comment:
Afterthought: we probably should use `database == dataset` (and therefore remove the `database` option completely). 

This will make the relation between Prodigy and pyannote.database much clearer.